### ISSUE TITLE
QuantityType.heartRateVariabilitySDNN always empty

### ISSUE THREAD

**Issue by YasufumiMuranaka** [VictorKachalov/health_kit_reporter#9]
**Describe the bug**
QuantityType.heartRateVariabilitySDNN is always returned empty

**To Reproduce**

**Expected behavior**

<img width="1680" alt="スクリーンショット 2021-01-18 21 06 44" src="https://user-images.githubusercontent.com/33305925/104921988-6d389000-59dd-11eb-88b2-6dc45080ddad.png">

**Screenshots**

I could retrieve e.g. HEART_RATE but few QuantityType always empty.
<img width="963" alt="スクリーンショット 2021-01-18 22 29 13" src="https://user-images.githubusercontent.com/33305925/104922084-8e997c00-59dd-11eb-8e42-61071a427f53.png">

**Additional context**
iPhone 12 Pro Max iOS 14.3

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.5, on macOS 11.1 20C69 darwin-arm, locale ja-JP)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
[!] Android Studio (version 4.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.52.1)
[✓] Connected device (1 available)

! Doctor found issues in 1 category.
```

**Comment by YasufumiMuranaka** [comment]
@VictorKachalov 
Hi!
**I found this cause of problem.**

one sample found.
<img width="1302" alt="スクリーンショット 2021-01-20 12 14 43" src="https://user-images.githubusercontent.com/33305925/105123326-31a3e000-5b1b-11eb-98e2-86afdb13c091.png">

but parse exception was occurred.
<img width="1416" alt="スクリーンショット 2021-01-20 12 15 54" src="https://user-images.githubusercontent.com/33305925/105123349-3e283880-5b1b-11eb-8c33-ae19560ae426.png">

because do not handle about heartRateVariabilitySDNN.
<img width="1330" alt="スクリーンショット 2021-01-20 12 28 13" src="https://user-images.githubusercontent.com/33305925/105123368-4aac9100-5b1b-11eb-8cd6-34935246046a.png">

**Comment by YasufumiMuranaka** [comment]
I successfully got this when added following case.
```
case HKObjectType.quantityType(forIdentifier: .heartRateVariabilitySDNN):
    return quantity(unit: HKUnit.secondUnit(with: .milli))
```

**Comment by VictorKachalov** [comment]
Hi @YasufumiMuranaka 

Yes, I also investigated this issue, it comes from the original pod https://github.com/VictorKachalov/HealthKitReporter 
I made a new release 1.3.2 of that pod, providing appropriate fixes. I will also make changes in the documentation for this plugin, because usage of a sampleQuery is a low level of like quantitySampleQuery. The difference is that if sampleQuery is called for a QuantityType, the Unit for the value will be taken regarding to SI, if quantitySampleQuery is called, the Unit will be taken automatically based on localization.
